src/Symfony/Component/HttpKernel/Profiler/BaseMemcacheProfilerStorage.php

```diff
@@ -11,300 +11,14 @@
 
 namespace Symfony\Component\HttpKernel\Profiler;
 
+use Symfony\Component\Profiler\Storage\BaseMemcacheProfilerStorage as Base;
```

Comment:
Was thinking about that but found the name BaseBase a bit weird.

Would probably be better if I change BaseMemcacheProfilerStorage to AbstractMemcacheProfilerStorage, this would then also follow the Naming Convensions